client/components/data/query-sites-media-storage/README.md

```diff
@@ -0,0 +1,8 @@
+Query Sites Media Storage
```

Comment:
In all the naming throughout these changes, is there a strong reason against eliminating "sites"? i.e.

`<QueryMediaStorage />`
`isRequestingMediaStorage`
`requestMediaStorage`
`receiveMediaStorage`